### PR TITLE
Embed video tag as live sample

### PR DESCRIPTION
**Pull request by peterbe**
This means we don't need to add CSP for that one single example.

### CODE REVIEW

**Comment by peterbe** [comment]
The video URL is still bad. We could change it to `https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm` which will work.

**Comment by Gregoor** [comment]
Thanks Peter, yeah I think also changing the video URL would be great. Do you want to do this in this PR or should I put it in a new one?

**Comment by peterbe** [comment]
> Thanks Peter, yeah I think also changing the video URL would be great. Do you want to do this in this PR or should I put it in a new one?

I'll change it to https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm

**Comment by peterbe** [comment]
Hmm.. I need your help. The old .webm was so small in width & height that it fit more nicely.

File: files/en-us/learn/html/cheatsheet/index.html [hunk 1]
```diff
@@ -171,9 +171,12 @@ <h2 id="Inline_elements">Inline elements</h2>
   </tr>
   <tr>
    <td>Embedded Video</td>
-   <td>&lt;video controls src="https://archive.org/download/WebmVp8Vorbis/webmvp8_512kb.mp4"&gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</td>
+   <td>&lt;video controls src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"&gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</td>
    <td>
-    <video controls src="https://archive.org/download/WebmVp8Vorbis/webmvp8_512kb.mp4">The <code>video</code> element is unsupported.</video>
+    <div class="hidden" id="sampleVideo">
+      <pre class="brush: html">&lt;video controls src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm"&gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</pre>
```
Comment (Gregoor) [inline files/en-us/learn/html/cheatsheet/index.html:177]:
```suggestion
      <pre class="brush: html">&lt;video controls src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm" style="width: 100%;" &gt;The &lt;code&gt;video&lt;/code&gt; element is unsupported.&lt;/video&gt;</pre>
```
that does the trick for me. Not perfect, but better than a dead link.

Btw the audio examples also seem to not work for me. Looks like this site needs some love in general

**Comment by peterbe** [comment]
I don't want to dwell on this page too much. But it's certainly true that the `<audio>` sample tags are weird but at least they don't crash into CSP :)

**Review by escattone — APPROVED** [review]
Looks and works great, thanks @peterbe!

**Pull request merged**